butia_emotions:

  eyebrown:

    standard:
      rightY: 50
      leftY: 50
      rightRotation: 130
      leftRotation: 150
      
    happy:
      rightY: 60
      leftY: 60
      rightRotation: 80
      leftRotation: 80

    sad:
      rightY: 60
      leftY: 50
      rightRotation: 170
      leftRotation: 120 

    rage:
      rightY: 30
      leftY: 50
      rightRotation: 110
      leftRotation: 172

    scared:
      rightY: 90
      leftY: 10
      rightRotation: 160
      leftRotation: 130

    doubt:
      rightY: 90
      leftY: 80
      rightRotation: 140
      leftRotation: 165

    sleepy:
      rightY: 100
      leftY: 0
      rightRotation: 130
      leftRotation: 150
    
  eyelid:

    standard:
      h: 70
      frequency: 0
      right: 50
      left: 50

    happy:
      h: 75
      frequency: 0
      right: 10
      left: 20

    sad:
      h: 60
      frequency: 0
      right: 55
      left: 55

    rage:
      h: 75
      frequency: 0
      right: 80
      left: 70

    scared:
      h: 60
      frequency: 0
      right: 0
      left: 10
    
    doubt:
      h: 80
      frequency: 0
      right: 60
      left: 50
    
    sleepy:
      h: 100
      frequency: 0
      right: 100
      left: 100

  neck:

    standard:
      horizontal: 180
      vertical: 180

    happy:
      horizontal: 180
      vertical: 180
    
    sad:
      horizontal: 180
      vertical: 180

    rage:
      horizontal: 180
      vertical: 180

    scared:
      horizontal: 180
      vertical: 180
    
    doubt:
      horizontal: 180
      vertical: 180
    
    sleepy:
      horizontal: 180
      vertical: 180

  eyes:

    standard:
      horizontal: 50
      vertical: 66

    happy:
      horizontal: 50
      vertical: 66

    sad:
      horizontal: 50
      vertical: 45

    rage:
      horizontal: 50
      vertical: 50

    scared:
      horizontal: 50
      vertical: 50
    
    doubt:
      horizontal: 50
      vertical: 88
    
    sleepy:
      horizontal: 65
      vertical: 50

  jaw:
  
    standard:
      left: 112
      right: 112

    happy:
      left: 108
      right: 108

    sad:
      left: 118
      right: 118

    rage:
      left: 118
      right: 118

    scared:
      left: 108
      right: 108
    
    doubt:
      left: 118
      right: 118
    
    sleepy:
      left: 118
      right: 118